pkgs/development/python-modules/unblob-native/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+  gitUpdater,
+  rustPlatform,
+  libiconv,
+}:
+
+buildPythonPackage rec {
+  pname = "unblob-native";
+  version = "0.1.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "onekey-sec";
+    repo = "unblob-native";
+    tag = "v${version}";
+    hash = "sha256-jpaBxKuQNfU0I3kCs67mM5dzGURSSHvqymhk43P7xXk";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit pname version src;
+    hash = "sha256-K2QTf4OlP4AH2JJiJ6r8PRkInSOQdIBQcSvY5tWr4mw";
+  };
+
+  nativeBuildInputs = with rustPlatform; [
+    maturinBuildHook
+    cargoSetupHook
+  ];
+
+  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [ libiconv ];
+
+  pythonImportsCheck = [ "unblob_native" ];
+
+  passthru.updateScript = gitUpdater { rev-prefix = "v"; };
```

Comment:
This doesn't update the `cargoDeps` hash

Reply:
I hope `nix-update-script` is the right solution :thinking:

Reply:
Yes, it works now, thanks!

By the way to test the update script you can downgrade the version (doesn't matter if it doesn't exist), change the hashes (fails with empty has, so usually I just replace a few characters with zeros) then run `nix-shell maintainers/scripts/update.nix --argstr package python312Packages.unblob-native`. After the script finishes you should see the version and hashes changed back

Reply:
Yep, did the same to test it :relieved: